Src/BootCamp.Chapter/Player.cs

```diff
@@ -29,79 +30,142 @@ public class Player
         /// Player items. There can be multiple of items with same name.
         /// </summary>
         private Inventory _inventory;
+
         /// <summary>
         /// Needed only for the extra task.
         /// </summary>
         private Equipment _equipment;
```

Comment:
`_equipment` is never initialised so trying to equip an item raises a `NullReferenceException`

Reply:
added.